wgpu-core/src/conv.rs

```diff
@@ -383,6 +384,7 @@ pub(crate) fn map_texture_format(
                 H::D32SfloatS8Uint
             }
         }
+        Tf::Stencil8 => H::S8Uint,
```

Comment:
We need this conditional on what the adapter supports. See how it's done for `Depth24Plus`. Here, we'd need even more choices: it will be `S8Uint`, or `D24UnormS8Uint`, or `D32SfloatS8Uint`.

Reply:
Sure thing, will make this change. I'm probably going to be predisposed until this weekend, sorry for the delay.